src/ripple/app/misc/NetworkOPs.cpp

```diff
@@ -2021,6 +2023,30 @@ Json::Value NetworkOPsImp::getServerInfo (bool human, bool admin)
     {
         if (app_.config().VALIDATION_PUB.size ())
         {
+            auto const& validation_manifest =
+                app_.config().section (SECTION_VALIDATION_MANIFEST);
+
+            if (! validation_manifest.lines().empty())
+            {
+                std::string s;
+                s.reserve (188);
+                for (auto const& line : validation_manifest.lines())
+                    s += beast::rfc2616::trim(line);
+                if (auto mo = make_Manifest (beast::detail::base64_decode(s)))
+                {
+                    Json::Value valManifest = Json::objectValue;
+                    valManifest [jss::master_key] = toBase58 (
```

Comment:
I'd prefer it be base58 since this is the key that they would publish/share for others to add to the trusted validators list.